### PR TITLE
Clean up: service link

### PR DESCRIPTION
**Pull request by UlrikeS91**
MINOR changes:
- improved instructions
- renamed `name` to `displayLabel` since this is what we are actually asking for (which is describe din the instructions)

MAJOR changes:
none

SPECIAL ATTENTION:
- replaced embedded URL for `openDataIn` by IRI; this is the only case were URL is embedded which is not very consistent, IRI achieves almost the same thing and is also "embedded"

### CODE REVIEW

File: schemas/data/serviceLink.schema.tpl.json [hunk 1]
```diff
@@ -7,28 +7,29 @@
...
+    "openDataIn": {      
+      "type": "string",
+      "_formats": [
+        "iri"
+      ],
+      "_instruction": "Enter the internationalized resource identifier (IRI) to the linked data in the specified service."
     },
     "previewImage": {
-      "_instruction": "Add a (static) preview image file (URL) that is representative for this service link and could function as an icon.",
+      "_instruction": "Add an image (static or moving) to this service link that acts as a preview of its content or could function as an icon.",
```
Comment (lzehl) [inline schemas/data/serviceLink.schema.tpl.json]: "Add a file instance of an image or a small movie for this service link that can act as a preview or function as an icon."

**Review by lzehl — CHANGES_REQUESTED** [review]

**Review by lzehl — APPROVED** [review]
changed instructions for "previewImage" to "Add an image file to this service link ..." because if this can be a static image or a moving image (e.g. gif) is actually something the database system needs to define.

**Comment by lzehl** [comment]
@Peyman-N or @apdavison or @aeidi89 feel free to merge if you also approve the changes

**Review by aeidi89 — APPROVED** [review]
This looks good.

**Pull request merged**